### ISSUE TITLE
Https ui broken in Docker/Kubernetes

### ISSUE THREAD

**Issue by tikokito123** [grokability/snipe-it#11179]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

working with argocd (k8s) and nginx-controller, the domain name on aws route53 as same as the certificate.

When I open the ui as http request it is works fine but with the https the ui is broken, like it doesn't read the css files.

any ideas?

### Reproduction steps

1. added a certificate at aws acm
2. annotated it to nginx controller
3. open the web at https://my.domain.snipe and the ui is broken
...

### Expected behavior

To see the page (as normal) as the http shows me.

### Screenshots

![Screenshot from 2022-05-23 05-52-02](https://user-images.githubusercontent.com/30938070/169734746-e83e1785-6135-4189-a9b4-88e4ddc1dc3a.png)

### Snipe-IT Version

6.0.0

### Operating System

helm k8s alpine

### Web Server

n/a

### PHP Version

PHP 7.4.3 

### Error messages

```shell
No errors...
```

**Comment by snipe** [comment]
The UI is definitely not broken, but we'd need more info here. What do you have in your .env file for your APP_URL? What are the browser console messages?

**Comment by tikokito123** [comment]
> 

thanks for the reply,

APP_ENV=production

The logs on the browser show this error:
```
Blocked loading mixed active content “http:/mydomain/css/dist/all.css?id=b657d018489da88020dd”
```

```
Loading failed for the <script> with source “http://mydomain/js/dist/all.js?id=0c8d1423e5806757a9e4”.
```

**Comment by snipe** [comment]
I need APP_URL, not APP_ENV.

**Comment by tikokito123** [comment]
> I need APP_URL, not APP_ENV.

the app url is APP_URL=https://mydomain.com

<blockquote><div><strong><a href="https://mydomain.com/">Web Hosting, Domain Name Registration - MyDomain.com</a></strong></div><div>Small business web hosting offering additional business services such as: domain name registrations, email accounts, web services, and various small business solutions.</div></blockquote>

**Comment by k725** [comment]
Hi. I am running into the same problem.

I'm deploying to k8s and it's almost the same situation as him.
`APP_URL` is set for `https://snipe-it.example.com/`. When I specified different `APP_URL` the URL of `https://example.example.com/` was converted to http and displayed.
From this, I think `APP_URL` is loaded, but it seems that an unintended conversion is occurring.

I specified `APP_FORCE_TLS` to `true`, but apparently this doesn't work either.

**Comment by k725** [comment]
I don't think this is a good way to do it, but it's a workaround.

I tested on `snipe/snipe-it:v6.0.2-alpine`

```
$ kubectl exec -it $YOUR_DEPLOYMENT -- php upgrade.php
```

or

```
$ kubectl exec -it $YOUR_DEPLOYMENT -- composer upgrade
```

I think the cause of the dependency.

**Comment by nahsi** [comment]
Same issue running with Nomad.

![image](https://user-images.githubusercontent.com/28529663/173383067-9c0d3d2a-acd1-4ba2-af91-d3f887b8b091.png)
![image](https://user-images.githubusercontent.com/28529663/173383134-027a4212-8b78-4725-b21d-16c0659382ea.png)

APP_URL is set to `https://snipeit.service.consul`

**Comment by csabavirag** [comment]
I tried a couple of things like on the PHP-FPM image
- configure `APP_TRUSTED_PROXIES` in my env file
- add `APP_FORCE_TLS=true`

but nothing worked on Kubernetes at the end. Worked while running the `docker-php-entrypoint` script, but got broken when launched it via `docker-snipeit-entrypoint`.
 
So I tried to execute every command from `docker-snipeit-entrypoint.sh` step-by-step and realized, everything was OK until the `php artisan config:cache` step.
As Artisan documentation states, once the `config:cache` command is called, no more `.env` file parsing will happen, only `bootstrap/cache/config.php` file is used where the env variables are compiled into, but for example TLS enforcement check is done via the `env` function in AppServiceProvider class, so this `APP_FORCE_TLS` env variable seems to be discarded after `config:cache` command completely.

My final solution is to inject the `HTTPS` environment variable to my Kubernetes pod with the value of `on` (making it as a `$_SERVER` variable - like it's parsed in SettingsController) and everything works fine now. All the assets/links are created with https prefix now.
